runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -3318,101 +3264,83 @@ genTestIsSuper(TR::CodeGenerator * cg, TR::Node * node,
       bytesOffset = 2;
       }
 
-   if (dynamicCastClass)
+   TR::Register *toClassDepthReg = NULL;
+   TR::Register *fromClassDepthReg = NULL;
+   if (toClassDepth == -1)
       {
-      cursor = generateRXInstruction(cg, loadOp, node, scratch2Reg,
-            generateS390MemoryReference(castClassReg, offsetof(J9Class, classDepthAndFlags) + bytesOffset, cg), cursor);
+      toClassDepthReg = srm->findOrCreateScratchRegister();
+      if (comp->getOption(TR_TraceCG))
+               traceMsg(comp, "Scratch reg holding toClassDepth is: %s\n", toClassDepthReg->getRegisterName(comp));
```

Comment:
I do not thing this message holds any useful information for the trace. I would recommend using instruction comment to provide a brief info about what the instruction is doing.